tests/OmniSharp.Script.Tests/WorkspaceInformationTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Scripting.Hosting;
+using OmniSharp.Models.WorkspaceInformation;
+using TestUtility;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace OmniSharp.Script.Tests
+{
+    public class WorkspaceInformationTests : AbstractTestFixture
+    {
+        public WorkspaceInformationTests(ITestOutputHelper output)
+            : base(output)
+        {
+        }
+
+        [Fact]
+        public async Task SingleCsiScript()
+        {
+            using (var testProject = TestAssets.Instance.GetTestScript("SingleCsiScript"))
+            using (var host = CreateOmniSharpHost(testProject.Directory))
+            {
+                var workspaceInfo = await GetWorkspaceInfoAsync(host);
+                var scriptProjects = workspaceInfo.Projects.ToArray();
```

Comment:
Also, the `ToArray` is unnecessary if you just take the result of `Assert.Single`. Bonus!